docs/tutorials/hostpath/xfs_quota/prerequisites.md

```diff
@@ -0,0 +1,131 @@
+# Prerequisites for enabling XFS Quota on Hostpath LocalPV
```

Comment:
Can we link this prerequisites doc to the enable-xfs-quota.md prerequisite section somehow? Not sure whether that would a great idea. But the details seem to be distributed in various document although they should be but IMO linking them somehow would ease things up.

Reply:
The prerequisites doc's main sub-topics are linked to enable-xfs-quota.md's prerequisites pointers.

Reply:
Okay. Yeah! somehow missed it. That seems good.